randovania/games/dread/json_data/Ghavoran.txt

```diff
@@ -475,6 +475,8 @@ Extra - asset_id: collision_camera_007
               After Ghavoran - Burenia Transport Grapple Box or After Dairon - Power Switch 2
   > Event - Super Missile Rotatable Enky
       Destroy Enky
+  > Event - Super Missile Rotatable with Ledge Warp
+      Morph Ball and Ledge Warp (Intermediate)
```

Comment:
I would personally tack on Highly Dangerous Logic to this, since you can lock out the three checks below permanently, and one of those items could say be screw attack with the seed expecting you to fight golzuna to get into hanubia.

Reply:
A better example would be, among these three items, you have both gravity suit and a required grapple beam.

Reply:
It just crossed my mind that we could put an event in the early supers area, which can act as an alternative to highly dangerous logic.

Reply:
I'm thinking about this some more, and I'd actually rather prefer raising the trick difficulty to represent the danger involved with the trick. Ledge warps are an advanced enough trick that anyone enabling them in their preset will understand what they're getting into

Reply:
To be clear: I'd rather raise the trick difficulty than include highly dangerous logic here.

Reply:
The part that makes me not want to use a difficulty rating for the purpose of the dangerous action information is that there are other setups for ledge warps, which seems like a much better usage of difficulty levels than reasoning about dangerous logic.